tests/instruments/test_instrument.py

```diff
@@ -52,6 +125,28 @@ class Fake(Instrument):
     assert Fake.x.__doc__ == expected_doc
 
 
+def test_control_check_errors_get(generic):
+    generic.fake_ctrl_check_get_errors = True
+
+    def checking():
+        generic.error = True
+    generic.check_errors = checking
+    generic.fake_ctrl
+    assert generic.error is True
+    del generic.error
```

Comment:
Those dels at the end of tests seem superfluous?

Reply:
The generic object is created once (scope=module), therefore I have to delete that property afterwards, lest the next test still has generic.error == True.

Reply:
Ah, I missed that on the phone. Why the module scope though, should not be expensive to make a new instance for every test, right?